test/governance/GovernanceStaking.test.ts

```diff
@@ -880,4 +882,31 @@ describe("GovernanceStaking - staking with GD  and get Rewards in GDAO", () => {
   function rdiv(x: BigNumber, y: BigNumber) {
     return x.mul(BN.from("10").pow(27)).add(y.div(2)).div(y);
   }
+
+  it("it should transfer LP tokens of the contract and make use of decimals", async () => {
```

Comment:
this test doesnt test anything, because the only use of decimals is outside the contract in line 907 of your test